lib/shoulda/matchers/active_model/numericality_matchers/comparison_matcher.rb

```diff
@@ -86,19 +92,20 @@ def last_failing_submatcher
           end
 
           def submatchers
-            @_submatchers ||=
-              comparison_combos.map do |diff, submatcher_method_name|
-                matcher = __send__(submatcher_method_name, diff, nil)
-                matcher.with_message(@message, values: { count: @value })
-                matcher
-              end
+            @_submatchers ||= comparison_combos.map do |
```

Comment:
This seems like another change to please Rubocop but actually detracts from the readability, would you mind changing this back?

Reply:
Haha, no worries!